databases/week2/dbweek2.sql

```diff
@@ -0,0 +1,74 @@
+/* week 2 homework */ 
+
+/* working with tasks */ 
+
+USE hyf;
+SELECT * FROM task t ;
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id)
+VALUES ('study', 'do homework', '2022-02-11', '2022-02-11', '2022-02-15', 2, 4); 
+
+UPDATE task 
+SET title = 'its not all about money'
+WHERE id = 2;
+
+UPDATE task 
+SET due_date = '2017-12-25'
+WHERE id = 2;
+
+/* not sure what is the dif btwn Change a task status
+Mark a task as complete */
+
+DELETE FROM task 
+WHERE id = 15;
+
+
+/* Part 2: School database*/
+
+CREATE DATABASE school;
+DEFAULT CHARACTER SET = 'utf8mb4';
+USE school ;
+
+CREATE TABLE class (
+    id int(10) unsigned NOT NULL AUTO_INCREMENT,
+    name varchar(255) NOT NULL,
+    begins DATE NOT NULL,
+    ends DATE NULL DEFAULT NULL,
+    PRIMARY KEY (id)
+    )
+    
+    CREATE TABLE student (
+    id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name varchar(255) NOT NULL,
+    email varchar(255) NOT NULL,
+    phone  varchar(255) NULL,
+    class_id int(10) unsigned NOT NULL,
+ FOREIGN KEY (`class_id`) REFERENCES `class`(`id`) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
So if you delete a class with id=3, all students with class_id=3 will also be deleted. Is that ok with you? ;p